packages/core/src/components/tabs/tabs.tsx

```diff
@@ -12,99 +10,85 @@ import { TabsItem } from "./tabs.interfaces";
   shadow: true,
 })
 export class Tabs {
+  private mutationObserver?: MutationObserver;
+
   @Element()
   host!: HTMLDsoTabsElement;
 
-  private activeTab?: HTMLDsoTabElement;
-  private firstTab?: HTMLDsoTabElement;
-  private lastTab?: HTMLDsoTabElement;
-  private focussedTab?: HTMLDsoTabElement;
-
-  private get enabledTabs(): Array<{ tabsItem: TabsItem; element: HTMLDsoTabElement }> {
-    const tabElements = this.host.querySelectorAll("dso-tab");
-
-    if (tabElements) {
-      // Only enabled tabs (not disabled) are needed for setting focus via ArrowRight and ArrowLeft
-      return Array.from(tabElements).reduce((tabs: Array<{ tabsItem: TabsItem; element: HTMLDsoTabElement }>, tab) => {
+  private get enabledTabs() {
+    // Only enabled tabs (not disabled) are needed for setting focus via ArrowRight and ArrowLeft
+    return Array.from(this.host.querySelectorAll<HTMLDsoTabElement>(":scope > dso-tab")).reduce<HTMLDsoTabElement[]>(
+      (tabs, tab) => {
         if (!tab.disabled) {
-          tabs.push({
-            tabsItem: {
-              active: tab.active,
-            },
-            element: tab,
-          });
+          tabs.push(tab);
         }
+
         return tabs;
-      }, []);
-    }
+      },
+      [],
+    );
+  }
 
-    return [];
+  private get firstTab() {
+    return this.enabledTabs[0];
   }
 
-  private moveFocusToTab(tab?: HTMLDsoTabElement) {
+  private get lastTab() {
+    return this.enabledTabs[this.enabledTabs.length - 1];
+  }
+
+  private moveFocusToTab(tab: HTMLDsoTabElement | undefined) {
     tab?._dsoFocus();
-    this.focussedTab = tab;
   }
 
-  private moveFocusToPreviousTab(focussedTab?: HTMLDsoTabElement) {
-    const index = this.enabledTabs?.findIndex((tab) => tab.element === focussedTab);
+  private moveFocusToPreviousTab(focussedTab: HTMLElement) {
+    const index = this.enabledTabs.findIndex((tab) => tab === focussedTab);
+    const next = this.enabledTabs[index - 1] ?? this.lastTab;
```

Comment:
Niet previous?

Reply:
Had gekund. Het perspectief is "hier ben je nu, dit is de volgende tab".